samples/subsys/ipc/ipm_imx/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+    description: the sample reads the received data from the Messaging Unit
+        using the IPM and transmits it back unchanged
+    name: i.MX IPM sample
+tests:
+    test:
+        build_only: true
+        filter: CONFIG_SOC_IMX
```

Comment:
This should be filter: CONFIG_SOC_FAMILY_IMX

If we do that, do we need the platform_whitelist as well?

Reply:
Done - filter changed. The platform_whitelist is still needed, once @diegosueiro merges MU support into his work on i.MX7, platform_whitelist could be perhaps removed then.